src/Avalonia.Visuals/Media/Fonts/FontFamilyLoader.cs

```diff
@@ -20,21 +19,26 @@ static FontFamilyLoader()
 
         public static IEnumerable<Uri> LoadFontAssets(FontFamilyKey fontFamilyKey)
         {
-            return fontFamilyKey.FileName != null
-                ? GetFontAssetsByFileName(fontFamilyKey.Location, fontFamilyKey.FileName)
-                : GetFontAssetsByLocation(fontFamilyKey.Location);
+            if (fontFamilyKey.Source.OriginalString.Contains(".ttf")
```

Comment:
I think that will misbehave if for uris like
`/Assets/My.ttf.fonts/SourceSans#Source Sans Pro`

Reply:
EndsWith would be better 👍